app/controllers/cart_controller.rb

```diff
@@ -7,6 +7,7 @@ def add_item
   end
 
   def show
+    restrict_admin
```

Comment:
this keeps the admin from accessing the cart, wonder if we should have put it in a before_action: clause

Reply:
added to the questions list